tsdb/shard.go

```diff
@@ -472,6 +506,11 @@ func (s *Shard) SeriesKeys(opt influxql.IteratorOptions) (influxql.SeriesList, e
 		return []influxql.Series{{Aux: auxFields}}, nil
 	}
 
+	s.mu.RLock()
```

Comment:
I don't think you need "Is" in the function name. I think `Closed() bool` is cleaner. :bikeshed: